test.cpp

```diff
@@ -0,0 +1,73 @@
+/**
+    \brief The program implements a check of the function operation quadro_solve.cpp
+*/
+
+#include <stdio.h>
+#include <string.h>
+#include <ctype.h>
+#include "quadro_solve.cpp"
+
+/**
+ * @brief A funtion for reading values from terminal
```

Comment:
1) funtion --> function (please install a spell-checker for your code editor, I don't like mentioning typos)
2) What kind of values this function can read? 
3) I would rather move the whole while loop into this function and called it something like readCoefficients or something similar
```c++
while (inputVal(&a, &b, &c) != 3) {
        printf("Вы не ввели число\n");
        scanf("%*s");
}
```